item.rb

```diff
@@ -0,0 +1,25 @@
+# item classs
+class Item
+  def initialize(date)
+    @id = Random.rand(1..1000)
+    @genre = nil
+    @author = nil
+    @label = nil
+    @publish_date = date
+    @archived = move_to_archive(@publish_date)
+  end
+
+  def add_genre; end
+
+  def add_author; end
+
+  def add_label; end
+
+  def can_be_archived?(date)
+    (Date.today - Date.parse(date)).to_i > 10
+  end
```

Comment:
The method can_be_archived ? must be private.

Reply:
Hello, @Abenezer-Tilahun. can_be_archived? will be overridden later inside the child classes of the Item class. Is it possible to override a private method? if the answer is yes, I will make it a private method. I will check it.